### DISCUSSION TITLE
Reword "Recommended based on people you follow" when not following anyone

### DISCUSSION THREAD

**Discussion by ghost** [community/community#14803] [Other Feature Feedback, Questions, & Ideas]
The "Recommended based on people you follow" section should be reworded when a user does not follow anybody

![tofollow](https://user-images.githubusercontent.com/100533118/163027031-64f9d795-c685-4e2a-aa76-90eb9938632d.png)

**Comment by Tr1985** [comment]
> Phần "Được đề xuất dựa trên những người bạn theo dõi" nên được đổi từ khóa khi người dùng không theo dõi bất kỳ ai
> 
> ![tofollow](https://user-images.githubusercontent.com/100533118/163027031-64f9d795-c685-4e2a-aa76-90eb9938632d.png)
